skglm/experimental/quantile_regression.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from numba import float64
+from skglm.datafits import BaseDatafit
+from skglm.utils.prox_funcs import ST_vec
+
+
+class Pinball(BaseDatafit):
+    r"""Pinball datafit.
+
+    The datafit reads::
+
+        quantile * max(y - Xw, 0) + (1 - quantile) * max(Xw - y, 0)
```

Comment:
nitpick: the real value does not involve np.max, it is np.maximum(...). sum().
Maybe rewrite as a sum and use _i to denote sample indices ? check how sklearn does it

Reply:
I looked up [scikit-learn source code](https://github.com/scikit-learn/scikit-learn/blob/dc580a8ef5ee2a8aea80498388690e2213118efd/sklearn/linear_model/_quantile.py#L181), but they don't specify the expression.

I am more with the usage of ``sum`` and ``_`` to indicate samples.